### PR TITLE
Added workflow to build and publish docker image

### PR DESCRIPTION
**Pull request by jimthematrix**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - main
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: docker build -t ghcr.io/hyperledger/firefly:latest .
```
Comment (nguyer) [inline .github/workflows/docker.yml:15]:
```suggestion
        run: docker build -t ghcr.io/hyperledger/firefly-fabconnect:latest .
```

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+      - name: Push
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
+          docker push ghcr.io/hyperledger/firefly:latest
```
Comment (nguyer) [inline .github/workflows/docker.yml:20]:
```suggestion
          docker push ghcr.io/hyperledger/firefly-fabconnect:latest
```

**Review by nguyer — CHANGES_REQUESTED** [review]
Slight tweak neccesary

**Comment by jimthematrix** [comment]
thanks @nguyer for the suggested update, I had ethconnect and fabconnect next to each other in VSCode and I edited the docker.yml file in ethconnect! 😆

**Review by nguyer — APPROVED** [review]

**Pull request merged**